config/localization/en/app.json

```diff
@@ -2049,8 +2050,8 @@
       "VIEW_PROPOSAL": "View proposal"
     },
     "COMPLIANCE": {
-      "CLOSE_ONLY_MESSAGE": "Because you appear to be a resident of, or trading from, a jurisdiction that violates our terms of use, or have engaged in activity that violates our terms of use, you have been blocked. You have until {DATE} to withdraw your funds before your access to the frontend is blocked. If you believe there has been an error, please email {EMAIL}.",
-      "PERMANENTLY_BLOCKED_MESSAGE": "Because you appear to be a resident of, or trading from, a jurisdiction that violates our terms of use and previously have been given an opportunity to redress circumstances that led to restrictions on your account, you have been permanently blocked. If you believe there has been an error, please email {EMAIL}.",
+      "CLOSE_ONLY_MESSAGE": "Because you appear to be a resident of, or trading from, a jurisdiction that violates our terms of use, or have engaged in activity that violates our terms of use, you have been blocked. You have until {DATE} to withdraw your funds before your access to the frontend is blocked. If you believe there has been an error, please contact support at {HELP_LINK}.",
```

Comment:
since this is technically a breaking update, could we update the key as well (i.e. to `CLOSE_ONLY_MESSAGE_WITH_HELP` or something)? (helps avoid breaking v4-web unintentionally through a localization update)